### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (a[href$="examples"])

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2338]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Documentation - Microsoft MakeCode](https://makecode.microbit.org/docs)
#### Element path
li:nth-child(2) > a[href$="examples"]
#### Snippet
`<a rel="nofollow noopener" href="/examples">Examples</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.98 (foreground color: #4183c4, background color: #ffffff, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

Links in docs; seems to be fixed locally with some of the other changes, but I think the changes still need to be made to propagate to the live docs / css might need to be rebuilt?

**Comment by abchatra** [comment]
fixed
